core/commands/version.go

```diff
@@ -30,8 +30,8 @@ const (
 
 var VersionCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline:          "Show ipfs version information.",
-		ShortDescription: "Returns the current version of ipfs and exits.",
+		Tagline:          "Show IPFS version information.",
```

Comment:
Show or Shows? We need to decide in this and many other cases which we prefer:

- (It) Shows IPFS version information.

or
* Show IPFS version information.

Reply:
https://github.com/ipfs/go-ipfs/pull/8010#discussion_r604383280